### PR TITLE
OpenStackClient Controller

### PR DESCRIPTION
**Pull request by slagle**

### CODE REVIEW

File: pkg/operator/helpers.go [hunk 1]
```diff
@@ -253,6 +254,19 @@ func GetCSVBase(name, namespace, displayName, description, image, replaces strin
 				"storage_class": "host-nfs-storageclass",
 			},
 		},
+		map[string]interface{}{
+			"apiVersion": "openstackclients.controlplane.openstack.org",
```
Comment (dprince) [inline pkg/operator/helpers.go]: Change it to this perhaps? controlplane.openstack.org/v1beta1

**Comment by dprince** [comment]
@slagle you might also go ahead and add the openstackclient API to the visible-crds-list. That way we'll see it in the UI

File: pkg/operator/helpers.go (hunk 1 above)
Comment (slagle) [reply pkg/operator/helpers.go]: Done.

**Review by dprince — APPROVED** [review]

File: controllers/suite_test.go [hunk 2]
```diff
@@ -60,6 +60,9 @@ var _ = BeforeSuite(func(done Done) {
 	err = controlplanev1beta1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+err = controlplanev1beta1.AddToScheme(scheme.Scheme)
```
Comment (dprince) [inline controllers/suite_test.go]: Both these lines look to be a duplicate of the lines above so I think you can remove it.

**Comment by dprince** [comment]
@slagle one inline review. Other than that LGTM from me.

File: controllers/suite_test.go (hunk 2 above)
Comment (slagle) [reply controllers/suite_test.go]: weird, must be a bug in the sdk code generation. I removed them.

**Comment by slagle** [comment]
> @slagle one inline review. Other than that LGTM from me.

Fixed.

**Pull request merged**